src/lib/XMTPPush.ts

```diff
@@ -1,11 +1,17 @@
+import { Client } from './Client'
 import * as XMTPModule from '../index'
 
-export class XMTPPush {
+export default class XMTPPush {
```

Comment:
```suggestion
export class XMTPPush {
```

fix build error causing failed lint CI, etc

Reply:
if you have a default export in XMTPPush.ts, you need to import it into index.ts before re-exporting it. If it’s a named export, you can directly re-export it in index.ts without importing it first